useIsShake.js

```diff
@@ -69,15 +54,13 @@ export function useIsShake() {
   };
 
   useEffect(() => {
-    _subscribe();
-
-    const onLoad = setTimeout(() => {
-      isToggleReady = true;
-      clearTimeout(onLoad);
-    }, SVG_LOAD_DURATION);
-
+    // invocation of _subscribe when isToggleReady === true
+    if (isToggleReady) {
+      _subscribe();
+      // isShake Accelerometer listener is removed when !isToggleReady
+    } else _unsubscribe();
     return () => _unsubscribe();
-  }, []);
+  }, [isToggleReady]);
 
-  return { toggle };
+  return { isShakeTriggered, setIsToggleReady };
```

Comment:
`isShakeTriggered` is renamed state variable `toggle` and setIsToggleReady returned here additionally so that other components may invoke setter during animation completion callback logic.